modules/metadata/src/main/scala/com/snowplowanalytics/snowplow/scalatracker/metadata/package.scala

```diff
@@ -24,19 +24,15 @@ package object metadata {
      * Adds EC2 context to each sent event
      * Blocks event queue until either context resolved or timed out
      */
-    def enableEc2Context[G[_]: Sync](implicit F: Monad[F],
-                                     clock: ClockProvider[F],
-                                     uuid: UUIDProvider[F]): G[Tracker[F]] =
+    def enableEc2Context[G[_]: Sync](implicit F: Monad[F], clock: Clock[F], uuid: UUIDProvider[F]): G[Tracker[F]] =
```

Comment:
Minor: it seems that everywhere else we're using context bound syntax: `G[_]: Sync, F[_]: Monad: Clock: UUIDProvider]`. Also `F`, `clock` and `uuid` values are not used explicitly.

Reply:
This will not work without breaking the extends AnyVal.
It seemed that all the implicits counted as vals which is logical at some point. So this extender stays as AnyVal and the functions need to explicit request implicits, or the extender drops AnyVal but can work with the better looking syntax.
In my newly acquired knowledge of AnyVals I would stay with this not so pleasant syntax.